packages/server/lib/util/settings.js

```diff
@@ -3,7 +3,21 @@ const Promise = require('bluebird')
 const path = require('path')
 const errors = require('../errors')
 const log = require('../log')
-const { fs } = require('../util/fs')
+const { fs } = require('./fs')
+const requireAsync = require('./require_async').default
+const { CYPRESS_CONFIG_FILES } = require('../configFiles')
+const debug = require('debug')('cypress:server:settings')
+
+function jsCode (obj) {
+  const objJSON = obj && !_.isEmpty(obj)
+    ? JSON.stringify(_.omit(obj, 'configFile'), null, 2)
+    : `{
+
+}`
+
+  return `module.export = ${objJSON}
```

Comment:
Why didn't tests catch this?
```suggestion
  return `module.exports = ${objJSON}
```

Reply:
Why was it not caught? Because it is not run in the tests.

This code is used in #17000 to fill a default file that would be absent, the same way we would fill in cypress.json. Since this PR does not change the defaults, I thought it unnecessary to test something that is out fo scope.

Thank you for pointing the typo out though. I will commit it right away.